models/dodo/bnb/dodo_pools_bnb_trades.sql

```diff
@@ -0,0 +1,270 @@
+{{ config
+(
+    schema = "dodo_pools_bnb",
+    alias ='trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook='{{ expose_spells(\'["bnb"]\',
+                                    "project",
+                                    "dodo_pools",
+                                    \'["owen05"]\') }}'
+)
+}}
+    
+{% set project_start_date = '2021-02-22' %}
+
+{% set dodo_proxies = [
+"0x8E4842d0570c85Ba3805A9508Dce7C6A458359d0", 
+"0x0596908263Ef2724fBfBcAfA1c983FCD7a629038",
+"0x165BA87e882208100672b6C56f477eE42502c820",
+"0xAB623fBcaeb522046185051911209F5b2C2a2e1f"
```

Comment:
We may need to change these addresses to lowercased.
```suggestion
"0x8e4842d0570c85ba3805a9508dce7c6a458359d0", 
"0x0596908263ef2724fbfbcafa1c983fcd7a629038",
"0x165ba87e882208100672b6c56f477ee42502c820",
"0xab623fbcaeb522046185051911209f5b2c2a2e1f"
```